### DISCUSSION TITLE
Cards not displaying

### DISCUSSION THREAD

**Discussion by mselmasry** [matt8707/ha-fusion#529] [Q&A]
I have published a port to access the fusion dashboard through web and i am able to access it but some of the cards are either not showing or showing empty. Screenshots below:

In browser:
![Screenshot 2024-07-14 141339](https://github.com/user-attachments/assets/fcaac11a-d871-493c-81a7-ba68357e84a4)

Inside Home Assistant:
![Screenshot 2024-07-14 141347](https://github.com/user-attachments/assets/286e6e16-3a32-4531-9783-3f59117dc083)

**Comment by mselmasry** [comment]
A quick update ... I was accessing the URL through NGINX proxy manager, I tried using the IP and it worked fine.
Would appreciate any pointers though as to why i get this behavior with NPM

**Reply by matt8707** [reply]
https://github.com/matt8707/ha-fusion/issues/344
